sample-apps/monaibundle/main.py

```diff
@@ -58,10 +58,22 @@ def init_infers(self) -> Dict[str, InferTask]:
         #################################################
         # Models
         #################################################
+
         for n, b in self.models.items():
-            i = BundleInferTask(b, self.conf)
-            logger.info(f"+++ Adding Inferer:: {n} => {i}")
-            infers[n] = i
+            if "deepedit" in n:
+                # Adding automatic inferer
+                i = BundleInferTask(b, self.conf, type="segmentation")
+                logger.info(f"+++ Adding Inferer:: {n}_seg => {i}")
+                infers[n + "_seg"] = i
+                # Adding inferer for managing clicks
+                i = BundleInferTask(b, self.conf, type="deepedit", deepedit=True)
```

Comment:
you already have type="deepedit" then why you want to send deepedit=True again

Reply:
Absolutely right. PR updated. @SachidanandAlle